<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="leUrl">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>122.114.67.199</string>
      </property>
      <property name="placeholderText">
       <string>Url</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lePort">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>8000</string>
      </property>
      <property name="placeholderText">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tbConnect">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextBrowser" name="tbMessage">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
      <widget class="QWidget" name="widExec" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="leRemoteObject">
          <property name="placeholderText">
           <string>Remote Object</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="pbExec">
          <property name="text">
           <string>Exec</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Return</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leRemoteMethod">
          <property name="placeholderText">
           <string>Methods</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="pteRemoteMethodArgs">
          <property name="plainText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Args:[]</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leRemoteObject</tabstop>
  <tabstop>leRemoteMethod</tabstop>
  <tabstop>pteRemoteMethodArgs</tabstop>
  <tabstop>pbExec</tabstop>
  <tabstop>leUrl</tabstop>
  <tabstop>lePort</tabstop>
  <tabstop>tbConnect</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>tbConnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>switchConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbExec</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>callMethod()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>switchConnect()</slot>
  <slot>callMethod()</slot>
 </slots>
</ui>
